front/src/modules/rollingStock/components/RollingStockEditor/__tests__/RollingStockEditorForm.spec.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  getDefaultRollingStockMode,
+  getRollingStockEditorDefaultValues,
+  makeTractionMode,
+} from 'modules/rollingStock/helpers/utils';
+import {
+  DEFAULT_ELEC_POWER_STARTUP_TIME,
+  DEFAULT_RAISE_PANTOGRAPH_TIME,
+} from 'modules/rollingStock/consts';
+import { modifyRollingStockElectricalValues } from '../RollingStockEditorForm';
+
+const newEmptyRollingStock = getRollingStockEditorDefaultValues(null);
```

Comment:
```suggestion
const newEmptyRollingStock = getRollingStockEditorDefaultValues(null);
const electricRsEffortCurve = getDefaultRollingStockMode('1000');
const thermalRsEffortCurve = getDefaultRollingStockMode('thermal');
```
As you are using the same type of rollingstocks, we could maybe instantiate them outside the tests.

Reply:
done